src/newsList.js

```diff
@@ -0,0 +1,19 @@
+class NewsList {
+  constructor () {
+    this.list = []
+    this.counter = 0
+  }
+
+  createNewItem (headline, body, picture) {
+    this.list.push(new NewsItem(this.counter, headline, body, picture))
+    this.counter++
+  }
+
+  getList () {
+    return this.list
+  }
+
+  getCounter () {
+    return this.counter
+  }
+}
```

Comment:
news item and news list classes and functions, along with the tests all look good, no comments there.